metecho/api/tests/sf_run_flow.py

```diff
@@ -172,10 +172,12 @@ def test_mutate_scratch_org():
 
 
 def test_get_access_token():
-    with patch(f"{PATCH_ROOT}.SalesforceOAuth2") as SalesforceOAuth2:
+    with ExitStack() as stack:
+        OAuth2Client = stack.enter_context(patch(f"{PATCH_ROOT}.OAuth2Client"))
+        stack.enter_context(patch(f"{PATCH_ROOT}.OAuth2ClientConfig"))
         get_access_token(org_result=MagicMock(), scratch_org_config=MagicMock())
 
-        assert SalesforceOAuth2.called
+        assert OAuth2Client.called
```

Comment:
For new/reworked tests I've replaced context managers with `pytest-mocker` (which is already installed and enhances readability):

```python
def test_get_access_token(mocker):
    OAuth2Client = mocker.patch(f"{PATCH_ROOT}.OAuth2Client")
    mocker.patch(f"{PATCH_ROOT}.OAuth2ClientConfig")
    get_access_token(org_result=mocker.MagicMock(), scratch_org_config=mocker.MagicMock())

    assert OAuth2Client.called
```

At some point we should go back and apply the change to existing tests as well IMO